_notes/why_must_i_learn_this.md

```diff
@@ -0,0 +1,152 @@
+
+In real life, we have no time to learn things just-in-case.
+
+We need better reasons to learn something new!
+
+A better system to keep us going.
+
+In real life,
+
+if you want to fix your car tires, you pull up youtube to learn how to use a wrench.
+
+if you want to launch a new product, you learn how write more compelling titles.
+
+if you want to make your own perfume, you learn about the structures of aromatic compounds
+
+In real life, learning always takes place in the context of an application.
+
+“Why” we need this piece of knowledge comes BEFORE the actual learning.
+
+Learning is done just-in-time (JIT), not just-in-case (JIC)!
+
+I believe that the “why” is largely missing in today’s education system.
+
+Without it, the beauty of learning is lost.
+
+We leave the system less curious than before.
+
+That’s why I believe that the future of education needs to better address the “why”.
+
+# Two parters
+
+Understanding “why” has 2 parts.
+
+First, a student need to understand the context in which that knowledge was formed.
+
+For example, let’s look at housing policies concepts.
+
+What problems were the policy makers trying to solve?
+
+What tools did they have at hand?
+
+What were their hidden incentives?
+
+Half the battle is won just by understanding the context which the knowledge was formed.
+
+Second, the students need to understand how this relates to their world today.
+
+Application is possible if they draw similarities between the context of the past and the context of today.
+
+Only through applying the knowledge in a meaningful way is mastery of the knowledge truly achieved.
+
+# Poorly positioned
+
+Achieving the first part is not easy.
+
+Teachers that can do that well are hard to come by.
+
+Achieving the second part appears to be even more challenging for teachers.
+
+Why?
+
+The teachers themselves are not positioned to do the second part themselves!
+
+They do not spend extended periods of time applying the knowledge to solve real world problems.
+
+They do not have the chance to wrestle with all the fine nuances of how to manipulate that piece of knowledge.
+
+Therefore the appreciation for that knowledge is often not deep enough.
+
+The job of the school teacher is not to be the practitioner of the craft.
+
+We know this.
+
+That’s why business teachers invite successful founders to the classroom.
+
+That’s why physics teachers invite astronauts to the classroom.
+
+That’s why basketball coaches invite NBA players to the classroom.
+
+The practitioner’s job is to showcase how the learnings translate to real world application.
+
+Hopefully, this brings relevance to the material they are studying.
+
+Hopefully, this inspires the students to study better.
+
+# Untapped potential
+
+Why can’t we have 
+
+- more engineers give coding workshops to students?
+- more marketers give copywriting masterclass to students?
+- more designers give art lessons to students?
```

Comment:
I think we need to stress more on the perspective (contextXpractice) so its not "coding workshops" or "copywriting classes" that is stressed but instead the sharing of what perspectives can be brought into application by students

in-context with-persepctive